.github/workflows/deploy-ui.yaml

```diff
@@ -4,13 +4,30 @@ on: [push]
 jobs:
   build:
     runs-on: ubuntu-latest
-
+    env:
+      DEPLOY_TOKEN: ${{ secrets.CORD_19_DEPLOY }}
+    
     steps:
     - uses: actions/checkout@v2
     - name: Use Node.js 12
       uses: actions/setup-node@v1
       with:
         node-version: '12.x'
     - run: |
-        yarn install
-        yarn build
+        set -ex
+        cd "${GITHUB_WORKSPACE}"
+        ls -la
+        echo yarn install
+        echo yarn build
+        echo mv build ../build
+        git config user.name  "${GITHUB_ACTOR}"
+        git config user.email "${GITHUB_ACTOR}@verizonmedia.com"
```

Comment:
This won't work since github username isn't necessarily the same as VMG username?

Reply:
correct. we should have a generic email address here, suggestion? I will use info@vespa.ai just for fun for now